### PR TITLE
Ofmcc 1403 update doc upload ofm

### PR DESCRIPTION
**Pull request by jenbeckett**
UI updates and renaming of a heading

### CODE REVIEW

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/utils/constants.js [hunk 1]
```diff
@@ -197,7 +197,7 @@ export const SURVEY_QUESTION_TYPES = Object.freeze({
 })
 
 export const DOCUMENT_TYPES = Object.freeze({
-  FINANCIAL_STATEMENT: 'Financial Statement',
+  FINANCIAL_STATEMENT: 'Income Statement',
```
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/utils/constants.js:200]: done

File: frontend/src/utils/constants.js [hunk 2]
```diff
@@ -145,7 +145,7 @@ export const APPLICATION_ERROR_MESSAGES = Object.freeze({
   LICENCE_INFO: 'Licence information required',
   LICENCE_CONFIRMATION: 'Confirmation of licence information required',
   DOCUMENT_UPLOAD: 'Document upload required',
-  DOCUMENT_FINANCIAL_UPLOAD: 'Document upload for Financial Statement and Balance Sheet required',
+  DOCUMENT_INCOME_UPLOAD: 'Document upload for Income Statement and Balance Sheet required',
```
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/utils/constants.js:148]: done

File: frontend/src/views/applications/OperatingCostsView.vue [hunk 3]
```diff
@@ -35,44 +35,57 @@
       <v-col>
         <h4>Upload Documents</h4>
         {{ SUPPORTED_DOCUMENTS_MESSAGE }}
-        <AppMissingInfoError v-if="validation && !isFinancialDocsUploaded">{{ APPLICATION_ERROR_MESSAGES.DOCUMENT_FINANCIAL_UPLOAD }}</AppMissingInfoError>
+
```
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/views/applications/OperatingCostsView.vue:38]: done

File: frontend/src/utils/constants.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/utils/constants.js:200]: yes and done!

File: frontend/src/views/applications/OperatingCostsView.vue [hunk 4]
```diff
@@ -35,44 +35,57 @@
...
-          id="supporting-document-upload"
-          v-model="supporting.documentsToUpload"
-          entityName="ofm_applications"
-          :documentType="DOCUMENT_TYPES.SUPPORTING_DOCS"
-          :loading="processing"
-          :readonly="readonly"
-          :uploadedDocuments="supporting.uploadedDocuments"
-          @deleteUploadedDocument="deleteUploadedDocument"></AppDocumentUpload>
-      </v-card>
+      <h5>Financial Documents</h5>
```
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/views/applications/OperatingCostsView.vue:47]: perfect looks great :)

**Review by weskubo-cgi — APPROVED** [review]

**Pull request merged**